code/meta-parent/pom.xml

```diff
@@ -379,7 +379,7 @@
             <dependency>
                 <groupId>io.github.ibuildthecloud</groupId>
                 <artifactId>gdapi-java-server</artifactId>
-                <version>0.11.3</version>
+                <version>0.11.4-SNAPSHOT</version>
```

Comment:
Once we do a release of gdapi-java-server, I'll need to update this PR to be 0.11.4.
I wanted to get this cattle PR in for context to the gdapi-java-server change.